### ISSUE TITLE
Make QMPlay2 compatible with preview settings of Cat-Catch extension in chromium browsers 

### ISSUE THREAD

**Issue by trimechee** [zaps166/QMPlay2#780]
Hi, i can't watch streaming videos in my old weak eee pc, its i use preview setting of the amazing Cat-Catch extension to send streaming videos to potplyaer more light than heavy html5 web browser and there is no lag jerk in potplayer:

https://github.com/xifangczy/cat-catch

and i add this command recommended by the great developer of Cat-Catch extension in potplayer preview setting of Cat-Catch:

potplayer://${url} /referer="${referer}"

 also we should put this tool developed by great Cat-Catch developer in potplayer installation path to associate url protocol with potplayer:

https://github.com/xifangczy/URLProtocol

so preview with potplayer works perfectly until recently, chrome make modification in chromium custom url protocol mechanism and we can't preview video with potplayer anymore, i inform Mr Xifangczy about this capital bug and he informs me this bug is not related to Cat-Catch but related to potplayer that should fix this problem.....

so my idea is Mr @xifangczy add support of the amazing powerful QMPlay2 player to replace potplayer, and we please hope for a fruitful collaboration for the ecological interest of the planet between the great Mr @zaps166 and the great Mr @xifangczy to add support of the amazing super powerful QMPlay2 media player to preview settings of Cat-Catch, thanks!

i will mention my suggestion in the Cat-Caych github page, thanks :)

exemple de anime sites streaming pour tester preview settings : 

my 3 favourites anime streaming sites : 

https://v5.voiranime.com/?filter=dubbed

https://french-anime.com/animes-vf/

https://vostfree.ws/last-anime-vf.html

******

international sites : 

aniwave anix
https://aniwave.se/
aniwave.website
https://aniwave.se/official-app
https://9animetv.to/
https://movies2watch.tv/

https://www.reddit.com/r/Aniwave_se/
https://x.com/aniwave_se
https://www.reddit.com/r/aniwave/
https://www.reddit.com/r/9anime/

https://kissanime.com.ru/
https://kissanime.help/
https://kissasian.si/
https://kisswaifu.app/

https://allmanga.to/
https://github.com/pystardust/ani-cli
pystardust@protonmail.com

https://hianime.to/home
https://hianime.tv/
https://new.reddit.com/r/HiAnimeZone/
https://x.com/HiAnimeOfficial
https://t.me/HiAnimeLobby
https://discord.com/invite/hianime
https://www.reddit.com/r/HiAnimeZone/
https://t.me/HiAnimeLounge
https://twitter.com/HiAnimeOfficial
https://discord.gg/hianime
#hianime #aniwatch #zoroanime

https://animepahe.ru/
https://animepahe.org/
https://animepahe.com/
animepahe.ru 
animepahe.org 
animepahe.com

https://animixplay.to/
https://everythingmoe.com/
https://theindex.moe/
https://thewiki.moe/
https://fmhy.net/videopiracyguide#anime-streaming
https://www.reddit.com/r/animepiracy/
https://www.reddit.com/r/AroundAnimeTV/

https://animerush.in/

https://kawaiifu.com/
https://x.com/Kawaiifu_com

https://yugenanime.sx/
https://yugenanime.tv/
https://beta.yugenanime.tv/

gogoanime
https://gogoanimes.mobi/
https://anitaku.bz/home.html
https://s3embtaku.pro/
https://animedao.ac/
https://ww6.gogoanime2.org/
https://gogoanime.no/
http://gogoanime3.co/
https://w1.gogoanimehd.to/
https://www.gogoanime.pe/
https://gogotaku.info/
https://www.reddit.com/r/AroundAnimeTV/
https://gogoanime.co/
http://ww31.gogoanime.org/
https://discord.com/invite/kyVfcGuCCQ
https://discord.com/invite/gogo
https://new.reddit.com/user/KilluaZoldyck_0/

https://all-wish.me/
https://animesuge.pro/
https://about.me/animesugetv
https://www.reddit.com/r/AllwishOriginal/
https://t.me/allwishme
https://chat.whatsapp.com/CTdC085dEKSCDyYVNwcbql
animeshouse.live
https://animeshouse.quora.com/about
https://worldofonepiece.quora.com/
https://animeeshousee.blogspot.com/
https://www.facebook.com/animeeshouse/

https://animefox.sbs/home
https://x.com/animekisato
https://ko-fi.com/animesky

https://tv.zoro.se/home
https://zoroxtv.info/

https://anivibe.net/

https://anix.sh/home
https://www1.aniwave.ch/home

https://animixplay.name/

https://anicrush.tv/
https://anicrush.cc/
https://anicrush.to/
https://anicrush-to.com/
https://web.telegram.org/k/#@anicrushto
https://twitter.com/Anicrushto
https://discord.gg/btkpEZkD
(@AniCrush)

https://animehub.ac/

9anime aniwatch zoro

https://9animetv.to/
https://zoroxtv.info/
https://t.me/HiAnimeLobby
https://t.me/ZoroZone
https://www.reddit.com/r/HiAnimeZone/
https://web.telegram.org/k/#@ZoroZone
https://web.telegram.org/k/#-1944303479
https://t.me/HiAnimeFeeds
https://web.telegram.org/k/#@HiAnimeLobby

******************************************************************
lists liste best of

https://animixplay.to/

https://everythingmoe.com/#section-streaming
https://everythingmoe.com/
https://thewiki.moe/
https://www.reddit.com/r/StreamingAnime/wiki/topsites/
https://discord.com/invite/GuueaDgKdS
https://bsky.app/profile/everythingmoe.com
https://www.reddit.com/r/StreamingAnime/
https://www.reddit.com/r/everythingmoe/
https://www.reddit.com/user/thdr76/
https://discord.gg/snackbox
https://reddit.com/r/EverythingMoe
https://discord.gg/GuueaDgKdS

https://theindex.moe/
https://discord.com/invite/snackbox

https://github.com/justfoolingaround/animdl-provider-benchmarks
https://github.com/justfoolingaround/animdl?tab=readme-ov-file#honourable-mentions
https://github.com/justfoolingaround/animdl?tab=readme-ov-file#setting-up-providers
https://github.com/pystardust/ani-cli
https://github.com/CoolnsX/dra-cla
https://coolans.dev/
ansari.tan20@gmail.com
https://github.com/Lagradost/cloudstream-3
https://github.com/saikou-app/saikou
kr.justfoolingaround@gmail.com
https://discord.com/invite/gaX2Snst2j

https://github.com/Benexl/FastAnime
https://www.reddit.com/user/BenX855/
https://discord.gg/M98w9dDy
https://github.com/Benexl
http://benex254.github.io/
https://ko-fi.com/benex254
https://www.reddit.com/r/animepiracy/comments/1gudioo/do_you_really_need_a_gui_or_browser_to_watch_anime/
https://www.reddit.com/r/linux/comments/1gtjl0y/browse_anime_from_the_terminal/
https://www.reddit.com/r/commandline/comments/1grbcn7/comment/lydsn99/

https://www.reddit.com/r/animepiracy/
https://discord.com/invite/snackbox
seadex
https://releases.moe/
https://sheet.releases.moe/
https://discord.com/invite/jPeeZewWRn

******************************************************************

https://kuroiru.co/app
https://kuroiru.co/

https://www.vidbinge.com/
https://wafflehacker.io/
https://github.com/VidBingeApp/VidBinge
https://github.com/joshholly
https://www.instagram.com/wafflehacker
https://twitter.com/xss
https://www.vidbinge.com/
joshholly@gmail.com

https://movie-web-me.vercel.app/
https://not-movie-web.app/
https://www.reddit.com/r/Piracy/comments/18vm946/moviewebapp_is_everything_i_ever_wanted_in_a/?tl=fr
https://movie-web.github.io/docs/instances
https://www.reddit.com/user/BinaryOverload/
https://movie-web.github.io/
https://movie-web.app/
https://github.com/movie-web/movie-web/
https://movie-web.github.io/links/discord
https://instagram.com/vidbinge
https://facebook.com/vidbinge
https://discord.gg/4jzBvV8E5r

https://tv.kurr.dev/
https://github.com/kurrx/movie-web-app
https://www.linkedin.com/in/kurrx/
https://twitter.com/kurr_dev
http://kurr.dev/
ruslan@kurr.dev

https://netfilm.app/
https://netfilmapp.pics/
https://netfilm.city/
https://netfilm-app.com/
https://www.net-film.ru/en/

https://aniworld.to/
https://aniworld.domains/
https://onlinefilter.info/cuii-dns-sperre/serienstream-sto-dns-sperre-infos-details
https://s.to/
https://serien.domains/
https://onlinefilter.info/

https://kimcartoon.theproxy.app/
https://www.facebook.com/kimcartoonfp

https://www.tokyoinsider.com/
admin@tokyoinsider.com
tokyoinsider.com 
tokyoinsider.net 
tokyoinsider.org

https://nyaa.si/

**Comment by trimechee** [comment]
@Rtizer-9 Thanks for yout polite message but do you really think that the bosses of netflix, amazon prime, disnet monitor what a random user like me posts on a technical topic of a media player?!!

sorry but who are you @Rtizer-9 ? you are a co-developer of QMPlay2 ? if my post bothers Mr @zaps166 , Mr Zaps166  can delete it easily, there is no problem, but I will not delete anything because these sites help to solve the problem, I want to preview streaming video from these sites with qmplay2 so Mr Zaps166 should have the opportunity to test these sites, so I will not delete anything, because maybe it will be useful to Mr Zaps166 to do tests and make qmplay2 compatible with cat catch preview, if you are a co-developer of QMPLay2 and Mr @zaps166 authorizes you, then ok delete my post, github is full of streaming annuaries, do a search on github yourself, so it s legal to post sreaming sites t test them and github allow the post of streaming sites, , netflix disney does not need to stalk my post to find streaming sites, I posted them because these sites were again censored  and I had to do a huge colossal search to find the new links, so don't worry about netflix, netflix knows where to find these sites and my post does not change anything, to skip  wasting time Mr Zaps166  , i decide to post these sites, these sites are freely available on the internet, the rights holders have to delete them if they hope so, it is not my responsibility

**Comment by zaps166** [comment]
I'm lost (too much text).

TLDR: What's going on here?

**Comment by trimechee** [comment]
A user asked me to delete streaming anime websites , and i respnd these sites are important to test and add the feature preview with cat catch and qmplay2.....and Mr Błażej cn delete my post if he doesn't like that i post anime websites....

**Comment by zaps166** [comment]
I can delete whole this issue - do it @trimechee ?

**Comment by zaps166** [comment]
I decided to remove whole this issue. Pasting so many URLs for issue report is not a good idea and maybe some others might not want this.

1. I don't know what Cat-Catch is (too many text to read).
2. I don't use Chromium.
3. I don't want to add any URLs to the QMPlay2.

Maybe I can try to add passing custom HTTP headers somehow, but not in this issue report.

@trimechee I hope you'll receive an e-mail notification with my post. Please send confidential test URLs via e-mail.
